content/articles/implementing-flask-api-using-linked-list/index.md

```diff
@@ -0,0 +1,541 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /implementing-flask-api-using-linked-list-data-structure/
+title: Implementing Flask API Using Linked List Data Structure
+description: In this tutorial, we will discuss about Flask and linked lists. We will also learn how we can use linked list class to implement the core API endpoints.
+author: oruko-pius
+date: 2021-03-03T00:00:00-13:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/implementing-flask-api-using-linked-list-data-structure/hero.jpg
+    alt: Implementing Flask API Using Linked List Hero Image
+---
+
+### Introduction
+Understanding data structures is an important aspect when developing applications. As real-world applications consider space and time complexities when designing applications to save on the memory and time taken to run specific programs.
+<!--more-->
+
+Despite the useful trade-offs, they offer to the pool of developers, most rarely use them. These make their programs and software not being to the quality and standards expected and can make organizations incur huge losses in terms of budgeting for storage and consumer loading time of these applications.
```

Comment:
hello @OkothPius, could you kindly look at this paragraph? The second sentence is too long and also is hard to understand. Correct the sentence and punctuate it.
Then, try to explain the concept in simple terms for easy understanding.